### ISSUE TITLE
Two compatibility warnings in Foundry v10.291

### ISSUE THREAD

**Issue by bryancasler** [gsimon2/hide-player-ui#25]
When running Hide Player UI v1.5.1 on FoundryVTT v10.291 I get two warnings, these are the console messages:

```
[hide-player-ui] The module "hide-player-ui" contains "dependencies" which is deprecated in favor of "relationships.requires" setup.js:145 
[hide-player-ui] module "hide-player-ui" contains "system" which is deprecated in favor of "relationships.systems" setup.js:145 
```

![CleanShot 2023-01-21 at 21 35 54@2x](https://user-images.githubusercontent.com/662993/213898016-8a117864-56d2-4882-a31d-6490b556450a.png)

**Comment by gsimon2** [comment]
Thanks for the callout! 

This should be fixed in version `1.5.2` of the module.

**Comment by bryancasler** [comment]
That did it; no more warnings. Thank you!
